### PR TITLE
Update Infomaniak S3 profiles

### PR DESCRIPTION
**Pull request by kallioli**
Updating All S3 profiles for Infomaniak Object Storage

### CODE REVIEW

**Comment by dkocher** [comment]
Thanks for maintaining these profiles. Can you elaborate why you want to change the region and storage class options to default AWS values?

**Comment by kallioli** [comment]
Existing profiles were unable to download data stored on Infomaniak servers. It was possible to upload but not download.  After several tests with the S3 profile (Deprecated path style requests).cyberduckprofile which works on all infrastructures. I've made the necessary changes so that the profiles work correctly with the S3 protocol.

The first modification concerns "s3.bucket.virtualhost.disable", which is a necessary configuration.

For the deletion of the region, the S3 protocol stipulates that if no region is provided by the client, the value "us-east-1" is automatically used, which corresponds to the value provided by infomaniak on its clusters. The same applies to the storage class.

I hope I've answered your questions :)

**Comment by dkocher** [comment]
> The first modification concerns "s3.bucket.virtualhost.disable", which is a necessary configuration.

The setting `s3.bucket.virtualhost.disable=true` to disable virtual host style requests is already present and has not been changed.

File: Infomaniak Public Cloud 01 (S3).cyberduckprofile [hunk 1]
```diff
@@ -37,16 +37,8 @@
...
         <false/>
-        <key>Region</key>
-        <string>auto</string>
-        <key>Regions</key>
-        <array>
-            <string>auto</string>
-        </array>
         <key>Properties</key>
         <array>
-            <string>s3.storage.class.options=STANDARD</string>
```
Comment (dkocher) [inline Infomaniak Public Cloud 01 (S3).cyberduckprofile:48]: There is no need to remove this explicit configuration although the default [^1] is now the same.

[^1]: https://github.com/iterate-ch/cyberduck/blob/ffc47d9fcad1cd16b74724064cd2d9f99d1141d2/defaults/src/main/resources/default.properties#L311

File: Infomaniak Public Cloud 01 (S3).cyberduckprofile [hunk 2]
```diff
@@ -37,16 +37,8 @@
         <false/>
         <key>Anonymous Configurable</key>
         <false/>
-        <key>Region</key>
-        <string>auto</string>
```
Comment (dkocher) [inline Infomaniak Public Cloud 01 (S3).cyberduckprofile:41]: While we default to `us-east-1` it doesn't harm to explicitly specify it here.

File: Infomaniak Public Cloud 01 (S3).cyberduckprofile [hunk 3]
```diff
@@ -37,16 +37,8 @@
         <false/>
         <key>Anonymous Configurable</key>
         <false/>
-        <key>Region</key>
-        <string>auto</string>
-        <key>Regions</key>
-        <array>
-            <string>auto</string>
-        </array>
```
Comment (dkocher) [inline Infomaniak Public Cloud 01 (S3).cyberduckprofile:45]: It is fine to omit these if the default `Region` is set above.

**Review by dkocher — CHANGES_REQUESTED** [review]

File: Infomaniak Public Cloud 01 (S3).cyberduckprofile (hunk 1 above)
Comment (kallioli) [reply Infomaniak Public Cloud 01 (S3).cyberduckprofile:48]: I've add the option in all profile

File: Infomaniak Public Cloud 01 (S3).cyberduckprofile (hunk 3 above)
Comment (kallioli) [reply Infomaniak Public Cloud 01 (S3).cyberduckprofile:45]: I've delete this part from all profiles

File: Infomaniak Public Cloud 01 (S3).cyberduckprofile (hunk 2 above)
Comment (kallioli) [reply Infomaniak Public Cloud 01 (S3).cyberduckprofile:41]: Add in all profiles except in "Infomaniak Public Cloud 01 (S3)" when this option is set in this profile. Bucket listing doesn't work.

**Comment by dkocher** [comment]
I would like to give _Infomaniak Public Cloud 01 (S3).cyberduckprofile_ a spin to have a look at the issue with a default region specified but cannot seem to find out where to find access credentials in the Infomaniak Manager. The documentation [^1] does only seem to cover OpenStack Swift profiles. I would appreciate if you can open a documentation pull request covering this profile in our repository [^2]

[^1]: https://www.infomaniak.com/en/support/faq/2282/swift-backup-with-cyberduck-macoswindows 
[^2]: https://github.com/iterate-ch/docs/blob/8e6849f6582c9c7838babf11d222264d7d5612b6/protocols/openstack/infomaniak.md

**Comment by kallioli** [comment]
Hello, for the EC2 credentials you need to create them in openstack cli with a public cloud account.

You can find the specific documentation [here](https://docs.infomaniak.cloud/documentation/04.object-storage/010.s3/) for S3 Protocol.

I'll create a dedicated section in your documentation

**Review by dkocher — APPROVED** [review]

**Pull request merged**